client/src/App.jsx

```diff
@@ -6,8 +6,8 @@ import { Wine } from "./components/menu/Wine";
 import { Rooms } from "./components/rooms/Rooms";
 import { Reservation } from "./components/reservations/Reservation";
 import { ReservationDetails } from "./components/reservations/ReservationDetails";
-import { Admin } from "./components/admin/Admin";
-import { AllReservations } from "./components/admin/AdminReservations";
+import { AdminLogin } from "./components/admin/AdminLogin";
+import { AllReservations } from "./components/admin/Admin.jsx";
```

Comment:
No need for .jsx in imports. Es6+ imports work without file extensions

Reply:
Done